apps/librelingo_json_export/tests/test_integration.py

```diff
@@ -10,15 +10,15 @@
 
 
 def read_json_file(path):
-    with open(path) as fp:
-        return json.load(fp)
+    with open(path) as f_p:
```

Comment:
elsewhere `f_p` was instead named `file` which I think makes more sense,  I think you could name all of them `file` or `file_pointer` :thinking: